external/anomaly/anomaly_classification/openvino.py

```diff
@@ -231,3 +255,78 @@ def __load_weights(path: str, output_model: ModelEntity, key: str) -> None:
         """
         with open(path, "rb") as file:
             output_model.set_data(key, file.read())
+
+    def _get_openvino_configuration(self) -> Dict[str, Any]:
+        """Return configuration required by the exported model."""
+        # This always assumes that threshold is available in the task environment's model
+        # cast is used to placate mypy
+        configuration = {
+            "threshold": cast(float, struct.unpack("f", (self.task_environment.model.get_data("threshold")))[0]),
+            "labels": [self.normal_label.name, self.anomalous_label.name],
+        }
+        if "transforms" not in self.config.keys():
+            configuration["mean_values"] = list(np.array([0.485, 0.456, 0.406]) * 255)
+            # TODO not sure why this is scale in original PR
+            # https://github.com/openvinotoolkit/anomalib/blob/ak/model_api/anomalib/core/model/inference.py#L229
```

Comment:
This comment is a placeholder for question. @akorobeinikov maybe you can explain this (minor issue but just for clarity).

Reply:
As I know in anomaly using transforms as preprocessing. It is the equivalent from modelAPI, but modelAPI working with images in range [0, 255]. So I use scale to make correct preprocessing in modelAPI wrapper.

Reply:
As I know in anomaly using transforms as preprocessing. It is the equivalent from modelAPI, but modelAPI working with images in range [0, 255]. So I use scale to make correct preprocessing in modelAPI wrapper

Reply:
Alright. Thanks for explaining!

Reply:
I guess we could remove TODO from the comments.